schema/schema.go

```diff
@@ -43,6 +45,8 @@ type TableColumn struct {
 	IsUnsigned bool
 	EnumValues []string
 	SetValues  []string
+	FixedSize  uint
```

Comment:
I see mostly FixedSize = MaxSize, can we just use one size?

Reply:
`MaxSize` is literally the maximum amount of data (e.g., characters in a varchar) that you can store. For example, for a `varchar(40)` it is `40`

`FixedSize`, on the other hand, is the number of characters that are stored by mysql - for example for a `char(40)` it is `40`.

In an earlier revision, I just had a `Size` property, but that does not help in my case: I also need to have a flag that indicates whether that size is variable or fixed.

Thus, since I did not want to limit how this property can be used, I decided against a boolean flag of what the size means and simply extract the size in both ways, although only 1 applies (similar to how we have `EnumValues` versus `SetValues`)